sdk/formrecognizer/Azure.AI.FormRecognizer/src/Constants.cs

```diff
@@ -1,16 +1,35 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+using System;
 
 namespace Azure.AI.FormRecognizer
 {
     internal static class Constants
     {
+        private const string AzurePublicCloud = "https://login.microsoftonline.com/";
+        private const string AzureChina = "https://login.microsoftonline.cn/";
+        private const string AzureGovernment = "https://login.microsoftonline.us/";
+
         public const string AuthorizationHeader = "Ocp-Apim-Subscription-Key";
 
         public const string OperationLocationHeader = "Operation-Location";
 
-        public const string DefaultCognitiveScope = "https://cognitiveservices.azure.com/.default";
+        public static string DefaultCognitiveScope = GetDefaultCognitiveScope(Environment.GetEnvironmentVariable("AZURE_AUTHORITY_HOST"));
```

Comment:
This isn't a reliable way to determine which cloud is being targeted. `AuthorityHost` can be set via code not just this environment variable. Can we not determine which cloud the client is targeting from the service endpoint? If not we will probably need to add something to the client options